CFC_DataCollector/tests/tripGenTests/TestTourModel.py

```diff
@@ -0,0 +1,92 @@
+import sys
+sys.path.append("./../../trip_generator")
+import unittest
+from tour_model import Location, TourModel
+
+class TestTourModel(unittest.TestCase):
+
+    def test_something(self):
+        """ Basic Sanity Check """
+        home = Location("home")
```

Comment:
How do you specify the times_allowed and the counts for the locations you are creating here?